client/components/advanced-credentials/style.scss

```diff
@@ -65,7 +65,7 @@
 .connection-status__disconnected,
 .connection-status__loading {
 	display: flex;
-	*:not(first-child) {
+	*:not(:first-child) {
 		margin-left: 8px;
 	}
```

Comment:
Maybe this change will affect the styles? Should we just remove that because it is not a valid CSS and it doesn't do anything right now? If we correct it, it may change the current behavior. I think the approach is to retain the styles in `trunk`.

Reply:
Thanks for looking! I tested locally and it would be broken if we removed the CSS. Right now, in trunk, although the colon is missing, it's still rendering properly ('Connected' text next to green circle): 

<img width="1146" alt="Screen Shot 2022-09-27 at 6 22 31 PM" src="https://user-images.githubusercontent.com/35543432/192667398-4d0f1d95-1703-43f9-aea1-456db2fbdcc8.png">

If we remove the CSS which adds a left margin, 'Connected' look like this instead: 

<img width="151" alt="Screen Shot 2022-09-27 at 6 21 01 PM" src="https://user-images.githubusercontent.com/35543432/192667420-c1326f22-f63c-4a8c-9fa9-563842d1a935.png">

So in this case, it would be best to fix it.

Reply:
Great! Thanks